sql/clients_last_seen_v1.sql

```diff
@@ -8,8 +22,16 @@ WITH current_sample AS (
     submission_date_s3 = @submission_date
 ), previous AS (
   SELECT
+    -- Drop last_date values after 28 days
```

Comment:
this is necessary to ensure `sql/clients_last_seen_v1.init.sql` only needs to scan 28 days

Reply:
I was assuming we'd let these drift beyond 28 days. I'd prefer to instruct users to always specify `COUNTIF(... 28 DAY)` rather than rely on implicit behavior of these fields becoming null past 28 days. If MAU calculations really are the overwhelming use case for this datset, though, perhaps the extra ergonomics of being able to do a straight `COUNT(...)` are worth it.

Reply:
this isn't about doing `count()` vs `countif()`, it's about making sure that you only need the last 28 days to get a consistent result